include/glow/Graph/Log.h

```diff
@@ -58,11 +81,13 @@ class LogContext {
   /// Dumps the log into a file named after the given \p funcName.
   void dumpLog(llvm::StringRef funcName);
 
-  /// Logs the node creation with a list of input nodes.
-  void logNodeCreation(const Node *newNode);
+  /// Logs the node creation. Also logs into Module log context if \p
+  /// logIntoModule set as true.
+  void logNodeCreation(const Node *newNode, bool logIntoModule = false);
```

Comment:
nit: You use a `Node *` here, but a `Node &` in `logNodeDeletion()`. Would be good to make consistent, probably making both a reference if possible.